sdk/storage/azure-storage-blob/pom.xml

```diff
@@ -76,7 +76,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.27.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
+      <version>1.28.0-beta.1</version> <!-- {x-version-update;unreleased_com.azure:azure-core;dependency} -->
```

Comment:
this is going to block release? is there a workaround ?

Reply:
I'll see if there is anything that can be done without changes outside of Storage code